src/seedu/addressbook/AddressBook.java

```diff
@@ -1150,8 +1157,8 @@ private static String getUsageInfoForExitCommand() {
      * @param sign  Parameter sign to be removed
      * @return  string without the sign
      */
-    private static String removePrefixSign(String s, String sign) {
-        return s.replace(sign, "");
+    private static String removePrefixSign(String s, String prefix) {
+        return s.replace(prefix, "");
```

Comment:
This method is not general enough to achieve the expected behaviour. Hint: prefix should be replaced only if it occurs at the beginning.